client/src/app/pages/controls/stakeholders/stakeholders.tsx

```diff
@@ -306,68 +116,244 @@ export const Stakeholders: React.FC = () => {
+            </ToolbarContent>
+          </Toolbar>
+          <TableComposable
+            {...tableProps}
+            isExpandable
+            aria-label="Stakeholders table"
+          >
+            <Thead>
+              <Tr>
+                <TableHeaderContentWithControls {...tableControls}>
+                  <Th {...getThProps({ columnKey: "name" })} />
+                  <Th {...getThProps({ columnKey: "displayName" })} />
+                  <Th {...getThProps({ columnKey: "jobFunction" })} />
+                  <Th {...getThProps({ columnKey: "groupCount" })} />
+                </TableHeaderContentWithControls>
+              </Tr>
+            </Thead>
+            <ConditionalTableBody
+              isLoading={isFetching}
+              isError={!!fetchError}
+              isNoData={stakeholders.length === 0}
+              numRenderedColumns={numRenderedColumns}
+            >
+              {currentPageItems?.map((stakeholder, rowIndex) => {
+                return (
+                  <Tbody
+                    key={stakeholder.id}
+                    isExpanded={isCellExpanded(stakeholder)}
+                  >
+                    <Tr>
+                      <TableRowContentWithControls
+                        {...tableControls}
+                        item={stakeholder}
+                        rowIndex={rowIndex}
+                      >
+                        <Td
+                          {...getExpandTdProps({ item: stakeholder, rowIndex })}
+                        />
+
+                        <Td width={25} {...getTdProps({ columnKey: "name" })}>
+                          {stakeholder.email}
+                        </Td>
+                        <Td
+                          width={10}
+                          {...getTdProps({ columnKey: "displayName" })}
+                        >
+                          {stakeholder.name}
+                        </Td>
+                        <Td
+                          width={10}
+                          {...getTdProps({ columnKey: "jobFunction" })}
+                        >
+                          {stakeholder.jobFunction?.name}
+                        </Td>
+                        <Td
+                          width={10}
+                          {...getTdProps({ columnKey: "groupCount" })}
+                        >
+                          {stakeholder.stakeholderGroups?.length}
+                        </Td>
+                        <Td width={20}>
+                          <AppTableActionButtons
+                            onEdit={() => editRow(stakeholder)}
+                            onDelete={() => deleteRow(stakeholder)}
+                          />
+                        </Td>
+                      </TableRowContentWithControls>
+                    </Tr>
+                    {isCellExpanded(stakeholder) ? (
+                      <Tr isExpanded>
+                        <Td />
```

Comment:
Oh I see, this is to space it out so it is indented past the toggle. I guess that makes sense... hm, it does kinda suck that we have to remember to put this extra Td in though. I wonder if it's worth adding a new component for `<ExpandedContentRowWithControls>` or something that renders the Tr, the two Tds with the right props, and then passes children in.

Probably not worth it for now, I'll make a note to maybe mess with that later.